src/main/java/com/revature/globetrotters/service/PostService.java

```diff
@@ -0,0 +1,77 @@
+package com.revature.globetrotters.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.revature.globetrotters.entity.Post;
+import com.revature.globetrotters.entity.PostComment;
+import com.revature.globetrotters.repository.PostCommentRepository;
+import com.revature.globetrotters.repository.PostRepository;
+
+@Service
+public class PostService {
+    @Autowired
+    private PostRepository postRepository;
+
+    @Autowired
+    private PostCommentRepository postCommentRepository;
+    
+    public Post createPost(Post post) {
+        return postRepository.save(post);
+    }
+
+    // TODO: write custom postRepository method
+    public List<Post> findPostsByUserId(Integer userId) {
+        return postRepository.findByUserId(userId);
+    }
+
+    public Post findPostById(Integer postId) throws Exception {
+        Optional<Post> optionalPost = postRepository.findById(postId);
+       return optionalPost.orElseThrow(() -> new Exception("Post not found."));
+    }
+
+    public String deletePost(Integer postId) {
```

Comment:
We should check if a post with the postId given exists in the database first, and then delete it if it does. If it doesn't exist, we should throw an exception.